crates/rattler_installs_packages/src/utils.rs

```diff
@@ -84,5 +87,26 @@ pub fn normalize_index_url(mut url: Url) -> Url {
     url
 }
 
+/// Error that can occur while finding the python executable.
+#[derive(Debug, Error)]
+pub enum FindPythonError {
+    #[error("could not find python executable")]
+    NotFound,
+}
+
+/// Try to find the python executable in the current environment.
+pub fn python_executable() -> Result<PathBuf, FindPythonError> {
+    let python = which("python").map_err(|_| FindPythonError::NotFound);
+
+    // When installed with homebrew on macOS, the python3 executable is called `python3` instead
+    #[cfg(target_os = "macos")]
+    let python = if python.is_err() {
```

Comment:
I think this is not necessarily macOS specific. E.g. if you get Python from ubuntu or so, it can also be called `python3`.

Reply:
Maybe it would be better to look for `python3` and if that is not found try `python` instead of relying on the platform?

Reply:
Yeah, maybe? There is some discussion going on in the python feedstock: https://github.com/conda-forge/python-feedstock/issues/349

But I think for the pixi / conda case we will set this manually anyways so it doesn't really matter.

Reply:
Ok so I changed it to always detect it :)